packages/contracts-bedrock/src/cannon/MIPS2.sol

```diff
@@ -64,7 +64,7 @@ contract MIPS2 is ISemver {
 
     /// @notice The semantic version of the MIPS2 contract.
     /// @custom:semver 1.0.0-beta.25
```

Comment:
The natspec comment needs to be updated.

Reply:
Good catch, fixed in a060b94903.